packages/vuetify/src/composables/validation.ts

```diff
@@ -0,0 +1,80 @@
+// Utilities
+import { propsFactory, wrapInPromise } from '@/util'
+import { ref, unref, watch } from 'vue'
+
+// Types
+import type { MaybeRef } from '@/util/helpers'
+import type { PropType } from 'vue'
+
+export type ValidationRule = string | ((value: any) => string | boolean)
+export type ValidationRules = ValidationRule[]
+export type ValidationResult = string | boolean | Promise<string | boolean>
+
+export interface ValidationProps {
+  errorCount: string | number
```

Comment:
I found `errorCount` a big ambiguous in what it does, had to look at the code. Perhaps `maxErrors` or something similar is clearer?